README.md

```diff
@@ -53,7 +53,8 @@ and extending unit tests can be found in [/src/test/README.md](/src/test/README.
 
 There are also [regression and integration tests](/test), written
 in Python.
-These tests can be run (if the [test dependencies](/test) are installed) with: `test/functional/test_runner.py`
+These tests can be run (if the [test dependencies](/test) are installed) with: `build/test/functional/test_runner.py`
+(assuming `build` is your build directory).
```

Comment:
After [this push](https://github.com/bitcoin/bitcoin/compare/96f4820f1f274ad371aaa37f8247305183e4642c..cd9297c42447f41b95eea6794535a08341f479dd#diff-ec54487d37183db6e158a11eb3560af15758aba1bd7e253c6b3a58ad3ad16e39R13), ISTM that either the following change is now inadvertently missing, or the PR needs a new title.

```diff
-Running `test/functional/test_runner.py` with the `--coverage` argument tracks which RPCs are
+Running `build/test/functional/test_runner.py` (assuming `build` is your build
+directory) with the `--coverage` argument tracks which RPCs are
```